webapp/src/serialindicator.tsx

```diff
@@ -41,7 +41,7 @@ export class SerialIndicator extends React.Component<SerialIndicatorProps, Seria
         return(
             <div title={lf("Open console")} className="ui label circular" tabIndex={0} onClick={this.props.onClick} onKeyDown={sui.fireClickOnEnter}>
                 <div className="detail">
-                    <sui.Icon icon="bar graph"/>
+                    <img className="barcharticon" src={Util.pathJoin(pxt.webConfig.commitCdnUrl, `images/Bars_black.gif`)}></img>
```

Comment:
This image won't load in offline mode.

Reply:
How can we fix that?

Reply:
maybe we just hide it if it fails to load

Reply:
Open a bug to fix charting support when offline. This gif + smoothie need to be properly cached.